src/app/modules/form/public-form/public-form.utils.ts

```diff
@@ -110,3 +114,59 @@ export const mapFormAuthRedirectError: MapRouteError = (
       }
   }
 }
+
+/**
+ * Handler to map ApplicationErrors from validating a form's eServiceId to HTTP errors
+ * @param error The error to retrieve the status codes and error messages
+ * @param coreErrorMessage Any error message to return instead of the default core error message, if any
+ */
+export const mapValidateEsrvcIdError: MapRouteError = (
```

Comment:
is a totally separate error mapping for the validate endpoint needed? Seems to be the same as the auth redirect mapping. Should the two be combined, e.g. `mapFormAuthError`?

Reply:
i'm not too sure about this. my primary reasoning was because as per the previous one, maybe they are different enough to be separated (and exhaustive checking for cases). 

i'll merge them in if you feel it's too much!